libkbfs/folder_branch_ops.go

```diff
@@ -1055,8 +1055,14 @@ func (fbo *folderBranchOps) GetTLFCryptKeys(ctx context.Context,
 func (fbo *folderBranchOps) GetOrCreateRootNode(
 	ctx context.Context, h *TlfHandle, branch BranchName) (
 	node Node, ei EntryInfo, err error) {
-	err = errors.New("GetOrCreateRootNode is not supported by " +
-		"folderBranchOps")
+	err = errors.New("GetOrCreateRootNode is not supported by folderBranchOps")
```

Comment:
Sure, they used to be bare and tried to keep the same style - making them direct.